lib/install/validate/validator.go

```diff
@@ -902,3 +914,28 @@ func (v *Validator) syslog(op trace.Operation, conf *config.VirtualContainerHost
 		RAddr:   host,
 	}
 }
+
+// FIXME ATC DEBT setting this value needs to be moved to Dispatcher
+// https://github.com/vmware/vic/issues/6803
+// set PersistNetworkBacking key to "true"
+func (v *Validator) ConfigureVCenter(ctx context.Context) error {
+	op := trace.FromContext(ctx, "Set vCenter serial port backing")
+	defer trace.End(trace.Begin("", op))
+
+	errMsg := "Set vCenter settings SKIPPED"
+	if !v.sessionValid(op, errMsg) {
+		return nil
+	}
+	if !v.IsVC() {
+		op.Info(errMsg)
```

Comment:
I don't think we should even bother telling the user about things we didn't do that we shouldn't have done.
Same for earlier Info log in the same condition. Fine for it to be Debug.

Reply:
fixed